userguide/system.rst

```diff
@@ -2184,23 +2188,23 @@ feature requests.
 This screen provides a built-in interface to the %brand% issue
 tracker located at
 https://redmine.ixsystems.com/projects/freenas/issues.
-If you have not yet used the %brand% bug tracker, you must first go
-to that website, click the :guilabel:`Register` link, fill out the
+When using the %brand% bug tracker for the first time, go
+to the website, click the :guilabel:`Register` link, fill out the
 form, and reply to the registration email. This will create a username
 and password which can be used to create bug reports and receive
 notifications as the reports are actioned.
 
 Before creating a bug report or feature request, ensure that an
 existing report does not already exist at
 https://redmine.ixsystems.com/projects/freenas/issues.
-If you find a similar issue that is not yet marked as *closed* or
-*resolved*, add a comment to that issue if you have new information
-to provide that can assist in resolving the issue. If you find a
-similar issue that is marked as *closed* or *resolved*, you can
+If there is a similar issue that is not yet marked as *closed* or
+*resolved*, add a comment to that issue if there is new information
+to provide that can assist in resolving the issue. If there is a
+similar issue that is marked as *closed* or *resolved*,
 create a new issue and refer to the earlier issue number.
 
-.. note:: If you are not updated to the latest version of STABLE,
-   do that first to see if it resolves your issue.
+.. note:: Update the system to the latest version of STABLE,
+   first to see if that resolves the issue.
 
```

Comment:
Change the note to:
Always update the system to the latest version of STABLE to see if the issue is resolved prior to reporting a bug.